rules/math_big_rat.go

```diff
@@ -0,0 +1,50 @@
+package rules
+
+import (
+	"go/ast"
+
+	"github.com/securego/gosec/v2"
+)
+
+type usingOldMathBig struct {
+	gosec.MetaData
+}
+
+func (r *usingOldMathBig) ID() string {
+	return r.MetaData.ID
+}
+
+func (r *usingOldMathBig) Match(node ast.Node, ctx *gosec.Context) (gi *gosec.Issue, err error) {
+	callExpr, callExprOk := node.(*ast.CallExpr)
+	if !callExprOk {
+		return nil, nil
+	}
+
+	packageName, callName, callInfoOk := gosec.GetCallInfo(callExpr, ctx)
+	if callInfoOk != nil {
+		return nil, nil
+	}
+
+	if packageName != "math/big.Rat" || callName != "SetString" {
```

Comment:
Please could you the existing helper function to check this. Here is an example https://github.com/securego/gosec/blob/fb587c1d106addd2efc9d62c1ebc31a6caa9d746/rules/rsa.go#L35

Reply:
I had to fix this function a little, because it could not work with functions belonging to structures